test/performance/odp_queue_perf.c

```diff
@@ -25,13 +25,19 @@
 
```

Comment:
In commit message, is the mode handle in parentheses on purpose '-m 1' and not just '-m'?

Reply:
Improved commit message in V3.